test/prism/JsonUtilsTest.scala

```diff
@@ -27,20 +27,20 @@ class JsonUtilsTest
       val json = Json.parse(AMIs.validAMI)
       val ami = extractAMI(json).awaitEither.right.value
       ami should have(
-        'arn("arn:aws:ec2:region::image/ami-example"),
-        'name(Some("ami-name")),
-        'imageId("ami-example"),
-        'region("region"),
-        'description(Some("AMI description")),
-        'creationDate(Some(new DateTime(2016, 2, 11, 0, 0))),
-        'state("available"),
-        'architecture("x86_64"),
-        'ownerId("0123456789"),
-        'virtualizationType("hvm"),
-        'hypervisor("xen"),
-        'rootDeviceType("ebs"),
-        'sriovNetSupport(Some("simple")),
-        'upgrade(None)
+        Symbol("arn")("arn:aws:ec2:region::image/ami-example"),
```

Comment:
Nice one. We should migrate the API off Symbols at some point, but this is a good fix in the meantime 👍